lib/galaxy/dependencies/__init__.py

```diff
@@ -203,7 +203,7 @@ def check_total_perspective_vortex(self):
     def check_pbs_python(self):
         return "galaxy.jobs.runners.pbs:PBSJobRunner" in self.job_runners
 
-    def check_pykube(self):
+    def check_pykube_ng(self):
```

Comment:
Do we really need to rename this method?  I actually can't find any usages in the Galaxy codebase, so it likely doesn't matter, but this will break any external packages that call this method.

Reply:
This method is to support conditional dependencies. Galaxy will look for and call a method named `check_<package_name>` for each conditional dependency.